tests/phpunit/tests/query/lazyLoadTermMeta.php

```diff
@@ -15,6 +15,11 @@ class Test_Lazy_Load_Term_Meta extends WP_UnitTestCase {
 	 */
 	protected static $term_ids = array();
 
+	public function set_up() {
+		parent::set_up();
+		$this->reset_lazyload_queue();
+	}
+
```

Comment:
```suggestion
```

I don't think this is needed with the default reset in tearDown. Correct me if I am wrong.